scripts/squash-appdir.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+##==============================================================================
+##
+## This script squashes Mystikos application directories. It removes files that
+## are not necessary for execution within a Mystikos root file system.
+##
+## Example:
+##     squash-appdir.sh ./appdir
+##
+##==============================================================================
+
+if [ "$#" != "1" ]; then
+    echo "Usage: $0 <appdir-path>"
+    exit 1
+fi
+
+appdir=$(realpath $1)
+
+if [ "${appdir}" == "/" ]; then
+    echo "$0: refusing to squash the system root directory"
+    exit 1
+fi
+
+# Remove files with these extensions: .c, .h, .o
+rm -rf `find ${appdir} -name '*.[cho]'`
```

Comment:
I think we can also remove *.a files.

Reply:
Yes, I removed them and I was surprised that .NET actually checks dependencies on .a files and then fails.